prow/plank/reconciler.go

```diff
@@ -868,6 +886,13 @@ func pendingTriggeredIndexKeyByName(jobName string) string {
 	return fmt.Sprintf("pending-triggered-named-%s", jobName)
 }
 
+func pendingTriggeredIndexKeyByJobQueueName(jobQueueName string) string {
+	if jobQueueName == "" {
+		jobQueueName = "undefined"
```

Comment:
This would conflict with a user defined job queue with this name. We should just be able to use the empty string here, I don't think we need a magic value. If we do need a magic value for some reason, we should prevent users from using it themselves via config validation.

Reply:
Removed.